### DISCUSSION TITLE
Please add screensaver settings

### DISCUSSION THREAD

**Discussion by Silent-Hunter** [LizardByte/community#332] [Sunshine]
I use a CRT monitor (LCD monitors give me headaches and nausea), and Sunshine used to not mess with the Windows screensaver, but now it disables the screensaver with no option to enable it. I'd greatly appreciate an option to turn it back on.

**Comment by ReenigneArcher** [comment]
I do not think Sunshine does anything at all with screensaver options. No mention of it anywhere in the code or comments.

**Reply by Silent-Hunter** [reply]
Well that's weird, I wonder why it doesn't work? If I use VNC and leave it for 5 minutes, Windows turns on the screensaver, so it's probably something Sunshine or Moonlight is doing. Maybe it's sending something Windows is interpreting as user input. I'll try finding some sort of input monitor program and see if it shows anything.

**Reply by cgutman** [reply]
We do keep the display on while streaming to avoid issues with DXGI desktop duplication if the IDXGIOutput disappears while streaming. The goal isn't to suppress the screensaver but to avoid the display entering standby, however I'm not sure if there's any API that lets us allow one without the other.

**Reply by Silent-Hunter** [reply]
I see! Were you using something else before? In any case, if it's not fixable I may be able to find another solution, like making my own screensaver timer program.
